src/modules/components/proposal/proposalVoting/proposalVotingStage/proposalVotingStage.test.tsx

```diff
@@ -30,13 +36,43 @@ describe('<ProposalVotingStage /> component', () => {
         return <ProposalVotingStage {...completeProps} />;
     };
 
-    it('only renders the proposal status and the tabs for single-stage proposals', () => {
+    it('renders children directly when bodyList has one element', () => {
+        const bodyList = ['body1'];
+        const children = 'test-children';
+        render(createTestComponent({ bodyList, children }));
+        expect(screen.queryByTestId('proposal-body-summary')).not.toBeInTheDocument();
+        expect(screen.getByText(children)).toBeInTheDocument();
+    });
+
+    it('renders the children directly when bodyList is undefined and activeBody is null', () => {
+        const children = 'test-children';
+        render(createTestComponent({ bodyList: undefined, children }));
+        expect(screen.queryByTestId('proposal-body-summary')).not.toBeInTheDocument();
+        expect(screen.getByText(children)).toBeInTheDocument();
+    });
+
+    it('renders the proposal stage with its name inside an accordion item', () => {
```

Comment:
Looks like we already have this test below, maybe we should add a test that the component renders the stage name for single-stage and multi-body proposals